static/js/modules/columns.js

```diff
@@ -255,20 +257,18 @@ var filings = {
   },
   pdf_url: {
```

Comment:
I had this in mind but didn't get to it. I renamed to `document_type` to be generically descriptive.